v19.2/cockroachcloud-production-checklist.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Production Checklist
+summary: Learn how to move from testing to production on your CockroachCloud cluster.
+toc: true
+build_for: [cockroachcloud]
+---
+
+This page provides important recommendations for CockroachCloud production deployments.
+
+## Follow the SQL Best Practices
+
+To ensure optimal SQL performance for your CockroachCloud cluster, follow the best practices described in the [SQL Performance Best Practices](performance-best-practices-overview.html) guide.
+
+## Use a connection pool
```

Comment:
Maybe "Use a small pool of persistent connections" instead?

Reply:
That's my only nit. Otherwise this looks great. Feel free to merge once that's updated.

Reply:
Fixed.